### DISCUSSION TITLE
Prevent either Banner & Alert Notifications when sending out script.

### DISCUSSION THREAD

**Discussion by biggy726** [IBM/mac-ibm-notifications#210] [Q&A] [answered]
Hello,

First, I'd like to apologize if this is not the proper place to ask this. I'd also like to thank the contributors for all of their work in making it as easy for a novice (me)possible. After installing the Notifier app as well as the config profile and then send the script out, I get both a Banner and an Alert. I've tried everything I can think of, adjust the profile so only 1 type of Notification is allowed, adjust the script. This is not a dealbreaker, as I will still use it if this is not possible. There are some high-strung teachers in my District who will complain about even the smallest thing. Then again, maybe it is supposed to do this.

**Comment by SMartorelli** [comment]
Hi @biggy726 thanks for adopting IBM Notifier. 

When you say "I get both an alert and a banner" do you mean at the same time? Can you please send the script you're using (without any PI or sensitive informations of course) as well as the profile you're utilising for notification authorisation?

Thanks!

**Comment by biggy726** [comment]
Hello Simone,

First, thank you so much for replying to my rather vague question. What I
meant, more precisely is, the Banner will show up and as soon I click the
small "x" to close it then the Alert will show immediately after with the
choice of "ok" or "Close". However, I just discovered this morning that if
I just leave the banner only and allow it to go away by itself, then the
Alert does not show up. I'm sure it is something I'm doing wrong. Here are
the script and the profile I am using.

Again, I want to thank you and the entire team who have spent so much time
on this to make it easy for dummies like me.

Regards,
Robert

**Reply by SMartorelli** [reply]
Hi @biggy726 I don't see the attachments. Can you please attach it directly from Github? anyway based on your description make sure that in the script you use there aren't two sequential calls to IBM Notifier as it seems this the case.

**Comment by biggy726** [comment]
So sorry. Let me try again
[Robert's Notifier Script.txt](https://github.com/IBM/mac-ibm-notifications/files/14386119/Robert.s.Notifier.Script.txt)
[Fresh_Avoca_Notifier_Banner.txt](https://github.com/IBM/mac-ibm-notifications/files/14386120/Fresh_Avoca_Notifier_Banner.txt)

**Comment by SMartorelli** [comment, marked as answer]
Yes, as you can see in the script example there are two examples of notifier invocation in sequence.
If you delete the second one:
```
# Example 2 button prompt
RESPONSE=$(prompt_user "1")
echo "$RESPONSE"
```
only the first one will be triggered

**Comment by biggy726** [comment]
Thank you so much, Simone. I apologize for wasting your time with such an obvious, except to me, answer. I hope you have a great rest of the day.

Regards,
Robert

**Reply by SMartorelli** [reply]
Don't mention it, feel free to ask anytime :+1:
